commit_sha: 0f86cf095ae48524404070ef61223f2ad7f525ae  # The commit at which this recipe failed to build.
skiplist: false # Set to true to skiplist this recipe so that it will be ignored as long as its latest commit is the one given above.
log: |
  /opt/conda/lib/python3.8/site-packages/conda_build/cli/main_build.py:390: UserWarning: RECIPE_PATH received is a file (/opt/recipe/meta.yaml).
  It should be a path to a folder.
  Forcing conda-build to use the recipe file.
    warnings.warn(
  Updating build index: /opt/conda/conda-bld

  No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.21
  WARNING:conda_build.metadata:No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.21
  Adding in variants from internal_defaults
  INFO:conda_build.variants:Adding in variants from internal_defaults
  Adding in variants from /opt/host-conda-bld/conda_build_config_0_-e_conda_build_config.yaml
  INFO:conda_build.variants:Adding in variants from /opt/host-conda-bld/conda_build_config_0_-e_conda_build_config.yaml
  Adding in variants from /opt/host-conda-bld/conda_build_config_1_-e_bioconda_utils-conda_build_config.yaml
  INFO:conda_build.variants:Adding in variants from /opt/host-conda-bld/conda_build_config_1_-e_bioconda_utils-conda_build_config.yaml
  Attempting to finalize metadata for last
  INFO:conda_build.metadata:Attempting to finalize metadata for last
  [?25l[2K[0G[] 0.0s
  [2K[1A[2K[0Gfile:///opt/host-conda-bld/linux-64                129.0 B @   1.8MB/s  0.0s
  file:///opt/host-conda-bld/noarch                  127.0 B @   3.7MB/s  0.0s
  [] 0.1s
  conda-forge/linux-64  
  conda-forge/noarch    
  bioconda/linux-64     
  bioconda/noarch       
  pkgs/main/linux-64      [2K[1A[2K[1A[2K[1A[2K[1A[2K[1A[2K[0G[] 0.2s
  conda-forge/linux-64  
  conda-forge/noarch    
  bioconda/linux-64     
  bioconda/noarch       
  pkgs/main/linux-64      [2K[1A[2K[1A[2K[1A[2K[1A[2K[1A[2K[0G[] 0.3s
  conda-forge/linux-64  
  conda-forge/noarch    
  bioconda/linux-64     
  bioconda/noarch       
  pkgs/main/linux-64      [2K[1A[2K[1A[2K[1A[2K[1A[2K[1A[2K[0G[] 0.4s
  conda-forge/linux-64  
  conda-forge/noarch    
  bioconda/linux-64     
  bioconda/noarch       
  pkgs/main/linux-64      [2K[1A[2K[1A[2K[1A[2K[1A[2K[1A[2K[0G[] 0.5s
  conda-forge/linux-64  
  conda-forge/noarch    
  bioconda/linux-64     
  bioconda/noarch       
  pkgs/main/linux-64      [2K[1A[2K[1A[2K[1A[2K[1A[2K[1A[2K[0G[] 0.6s
  conda-forge/linux-64  
  conda-forge/noarch    
  bioconda/linux-64     
  bioconda/noarch       
  pkgs/main/linux-64      [2K[1A[2K[1A[2K[1A[2K[1A[2K[1A[2K[0G[] 0.7s
  conda-forge/linux-64  
  conda-forge/noarch    
  bioconda/linux-64     
  bioconda/noarch       
  pkgs/main/linux-64      [2K[1A[2K[1A[2K[1A[2K[1A[2K[1A[2K[0G[] 0.8s
  conda-forge/linux-64  
  conda-forge/noarch    
  bioconda/linux-64     
  bioconda/noarch       
  pkgs/main/linux-64      [2K[1A[2K[1A[2K[1A[2K[1A[2K[1A[2K[0G[] 0.9s
  conda-forge/linux-64  
  conda-forge/noarch    
  bioconda/linux-64     
  bioconda/noarch       
  pkgs/main/linux-64      [2K[1A[2K[1A[2K[1A[2K[1A[2K[1A[2K[0G[] 1.0s
  conda-forge/linux-64  
  conda-forge/noarch    
  bioconda/linux-64     
  bioconda/noarch      100%
  pkgs/main/linux-64    
  pkgs/main/noarch        [2K[1A[2K[1A[2K[1A[2K[1A[2K[1A[2K[1A[2K[0Gbioconda/noarch
  [] 1.1s
  conda-forge/linux-64  
  conda-forge/noarch    
  bioconda/linux-64     
  pkgs/main/linux-64    
  pkgs/main/noarch        [2K[1A[2K[1A[2K[1A[2K[1A[2K[1A[2K[0Gbioconda/linux-64
  [] 1.2s
  conda-forge/linux-64  
  conda-forge/noarch    
  pkgs/main/linux-64    
  pkgs/main/noarch      
  pkgs/r/linux-64         [2K[1A[2K[1A[2K[1A[2K[1A[2K[1A[2K[0G[] 1.3s
  conda-forge/linux-64  
  conda-forge/noarch    
  pkgs/main/noarch      
  pkgs/r/linux-64       
  pkgs/r/noarch           [2K[1A[2K[1A[2K[1A[2K[1A[2K[1A[2K[0G[] 1.4s
  conda-forge/linux-64  
  conda-forge/noarch    
  pkgs/main/noarch      
  pkgs/r/linux-64       
  pkgs/r/noarch           [2K[1A[2K[1A[2K[1A[2K[1A[2K[1A[2K[0G[] 1.5s
  conda-forge/linux-64  
  conda-forge/noarch    
  pkgs/main/noarch      
  pkgs/r/linux-64       
  pkgs/r/noarch           [2K[1A[2K[1A[2K[1A[2K[1A[2K[1A[2K[0Gpkgs/main/linux-64
  [] 1.6s
  conda-forge/linux-64  
  conda-forge/noarch    
  pkgs/main/noarch      
  pkgs/r/linux-64       
  pkgs/r/noarch           [2K[1A[2K[1A[2K[1A[2K[1A[2K[1A[2K[0G[] 1.7s
  conda-forge/linux-64  
  conda-forge/noarch    
  pkgs/main/noarch      
  pkgs/r/linux-64         [2K[1A[2K[1A[2K[1A[2K[1A[2K[0Gpkgs/r/noarch
  [] 1.8s
  conda-forge/linux-64  
  conda-forge/noarch    
  pkgs/main/noarch      
  pkgs/r/linux-64         [2K[1A[2K[1A[2K[1A[2K[1A[2K[0Gpkgs/main/noarch
  [] 1.9s
  conda-forge/linux-64  
  conda-forge/noarch    
  pkgs/r/linux-64         [2K[1A[2K[1A[2K[1A[2K[0G[] 2.0s
  conda-forge/linux-64  
  conda-forge/noarch    
  pkgs/r/linux-64         [2K[1A[2K[1A[2K[1A[2K[0Gpkgs/r/linux-64
  [] 2.1s
  conda-forge/linux-64  
  conda-forge/noarch      [2K[1A[2K[1A[2K[0G[] 2.2s
  conda-forge/linux-64  
  conda-forge/noarch      [2K[1A[2K[1A[2K[0G[] 2.3s
  conda-forge/linux-64  
  conda-forge/noarch      [2K[1A[2K[1A[2K[0G[] 2.4s
  conda-forge/linux-64  
  conda-forge/noarch      [2K[1A[2K[1A[2K[0G[] 2.5s
  conda-forge/linux-64  
  conda-forge/noarch      [2K[1A[2K[1A[2K[0G[] 2.6s
  conda-forge/linux-64  
  conda-forge/noarch   100%[2K[1A[2K[1A[2K[0Gconda-forge/noarch
  [] 2.7s
  conda-forge/linux-64    [2K[1A[2K[0G[] 2.8s
  conda-forge/linux-64    [2K[1A[2K[0G[] 2.9s
  conda-forge/linux-64    [2K[1A[2K[0G[] 3.0s
  conda-forge/linux-64    [2K[1A[2K[0G[] 3.1s
  conda-forge/linux-64    [2K[1A[2K[0G[] 3.2s
  conda-forge/linux-64    [2K[1A[2K[0G[] 3.3s
  conda-forge/linux-64    [2K[1A[2K[0G[] 3.4s
  conda-forge/linux-64    [2K[1A[2K[0G[] 3.5s
  conda-forge/linux-64    [2K[1A[2K[0G[] 3.6s
  conda-forge/linux-64    [2K[1A[2K[0G[] 3.7s
  conda-forge/linux-64    [2K[1A[2K[0G[] 3.8s
  conda-forge/linux-64    [2K[1A[2K[0G[] 3.9s
  conda-forge/linux-64    [2K[1A[2K[0G[] 4.0s
  conda-forge/linux-64    [2K[1A[2K[0G[] 4.1s
  conda-forge/linux-64    [2K[1A[2K[0G[] 4.2s
  conda-forge/linux-64    [2K[1A[2K[0G[] 4.3s
  conda-forge/linux-64    [2K[1A[2K[0G[] 4.4s
  conda-forge/linux-64    [2K[1A[2K[0G[] 4.5s
  conda-forge/linux-64    [2K[1A[2K[0G[] 4.6s
  conda-forge/linux-64    [2K[1A[2K[0G[] 4.7s
  conda-forge/linux-64    [2K[1A[2K[0G[] 4.8s
  conda-forge/linux-64    [2K[1A[2K[0G[] 4.9s
  conda-forge/linux-64    [2K[1A[2K[0G[] 5.0s
  conda-forge/linux-64    [2K[1A[2K[0G[] 5.1s
  conda-forge/linux-64    [2K[1A[2K[0G[] 5.2s
  conda-forge/linux-64    [2K[1A[2K[0G[] 5.3s
  conda-forge/linux-64    [2K[1A[2K[0G[] 5.4s
  conda-forge/linux-64    [2K[1A[2K[0G[] 5.5s
  conda-forge/linux-64    [2K[1A[2K[0G[] 5.6s
  conda-forge/linux-64    [2K[1A[2K[0G[] 5.7s
  conda-forge/linux-64    [2K[1A[2K[0G[] 5.8s
  conda-forge/linux-64    [2K[1A[2K[0G[] 5.9s
  conda-forge/linux-64    [2K[1A[2K[0G[] 6.0s
  conda-forge/linux-64    [2K[1A[2K[0G[] 6.1s
  conda-forge/linux-64    [2K[1A[2K[0G[] 6.2s
  conda-forge/linux-64    [2K[1A[2K[0G[] 6.3s
  conda-forge/linux-64 100%[2K[1A[2K[0G[] 6.4s
  conda-forge/linux-64 100%[2K[1A[2K[0G[] 6.5s
  conda-forge/linux-64 100%[2K[1A[2K[0G[] 6.6s
  conda-forge/linux-64 100%[2K[1A[2K[0G[] 6.7s
  conda-forge/linux-64 100%[2K[1A[2K[0G[] 6.8s
  conda-forge/linux-64 100%[2K[1A[2K[0G[] 6.9s
  conda-forge/linux-64 100%[2K[1A[2K[0G[] 7.0s
  conda-forge/linux-64 100%[2K[1A[2K[0G[] 7.1s
  conda-forge/linux-64 100%[2K[1A[2K[0G[] 7.2s
  conda-forge/linux-64 100%[2K[1A[2K[0G[] 7.3s
  conda-forge/linux-64 100%[2K[1A[2K[0Gconda-forge/linux-64
  [?25h[34mReloading output folder: [0m[34m/opt/conda/[0m[34mconda-bld[0m
  [?25l[2K[0G[] 0.0s
  [2K[1A[2K[0Gopt/conda/conda-bld/linux-64                        ??.?MB @  ??.?MB/s 0 failed  0.0s
  opt/conda/conda-bld/noarch                         127.0 B @   2.5MB/s  0.0s
  [?25h[34mReloading output folder: [0m[34m/opt/conda/[0m[34mconda-bld[0m
  [?25l[2K[0Gopt/conda/conda-bld/linux-64                       129.0 B @   2.1MB/s  0.0s
  [] 0.0s
  [2K[1A[2K[0Gopt/conda/conda-bld/noarch                         127.0 B @   2.9MB/s  0.0s
  [?25hBUILD START: ['last-490-hdbdd923_8.tar.bz2']
  [34mReloading output folder: [0m[34m/opt/conda/[0m[34mconda-bld[0m
  [?25l[2K[0G[] 0.0s
  [2K[1A[2K[0Gopt/conda/conda-bld/linux-64                       129.0 B @   1.7MB/s  0.0s
  opt/conda/conda-bld/noarch                         127.0 B @   4.5MB/s  0.0s
  [?25h
  ## Package Plan ##

    environment location: /opt/conda/conda-bld/last_1685259160896/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol


  The following NEW packages will be INSTALLED:

      _libgcc_mutex: 0.1-conda_forge    conda-forge
      _openmp_mutex: 4.5-2_gnu          conda-forge
      libgcc-ng:     12.2.0-h65d4601_19 conda-forge
      libgomp:       12.2.0-h65d4601_19 conda-forge
      libstdcxx-ng:  12.2.0-h46fd767_19 conda-forge

  Preparing transaction: ...working... done
  Verifying transaction: ...working... done
  Executing transaction: ...working... done
  [34mReloading output folder: [0m[34m/opt/conda/[0m[34mconda-bld[0m
  [?25l[2K[0G[] 0.0s
  [2K[1A[2K[0Gopt/conda/conda-bld/linux-64                       129.0 B @   2.9MB/s  0.0s
  opt/conda/conda-bld/noarch                         127.0 B @   4.5MB/s  0.0s
  [?25h[34mReloading output folder: [0m[34m/opt/conda/[0m[34mconda-bld[0m
  [?25l[2K[0G[] 0.0s
  [2K[1A[2K[0Gopt/conda/conda-bld/linux-64                       129.0 B @   2.6MB/s  0.0s
  opt/conda/conda-bld/noarch                         127.0 B @   4.5MB/s  0.0s
  [?25h
  ## Package Plan ##

    environment location: /opt/conda/conda-bld/last_1685259160896/_build_env


  The following NEW packages will be INSTALLED:

      _libgcc_mutex:            0.1-conda_forge    conda-forge
      _openmp_mutex:            4.5-2_gnu          conda-forge
      binutils_impl_linux-64:   2.39-he00db2b_1    conda-forge
      binutils_linux-64:        2.39-h5fc0e48_13   conda-forge
      gcc_impl_linux-64:        12.2.0-hcc96c02_19 conda-forge
      gcc_linux-64:             12.2.0-h4798a0e_13 conda-forge
      gxx_impl_linux-64:        12.2.0-hcc96c02_19 conda-forge
      gxx_linux-64:             12.2.0-hb41e900_13 conda-forge
      kernel-headers_linux-64:  2.6.32-he073ed8_15 conda-forge
      ld_impl_linux-64:         2.39-hcc3a1bd_1    conda-forge
      libgcc-devel_linux-64:    12.2.0-h3b97bd3_19 conda-forge
      libgcc-ng:                12.2.0-h65d4601_19 conda-forge
      libgomp:                  12.2.0-h65d4601_19 conda-forge
      libsanitizer:             12.2.0-h46fd767_19 conda-forge
      libstdcxx-devel_linux-64: 12.2.0-h3b97bd3_19 conda-forge
      libstdcxx-ng:             12.2.0-h46fd767_19 conda-forge
      make:                     4.3-hd18ef5c_1     conda-forge
      sysroot_linux-64:         2.12-he073ed8_15   conda-forge

  Preparing transaction: ...working... done
  Verifying transaction: ...working... done
  Executing transaction: ...working... done
  Source cache directory is: /opt/conda/conda-bld/src_cache
  INFO:conda_build.source:Source cache directory is: /opt/conda/conda-bld/src_cache
  Downloading source to cache: last-490_c4d600c373.zip
  INFO:conda_build.source:Downloading source to cache: last-490_c4d600c373.zip
  Downloading http://last.cbrc.jp/last-490.zip
  INFO:conda_build.source:Downloading http://last.cbrc.jp/last-490.zip
  WARNING:urllib3.connection:Certificate did not match expected hostname: last.cbrc.jp. Certificate: {'subject': ((('commonName', 'cbrc3.cbrc.jp'),),), 'issuer': ((('countryName', 'US'),), (('organizationName', "Let's Encrypt"),), (('commonName', 'R3'),)), 'version': 3, 'serialNumber': '046FF5BB109D36E9CD09D5B800EC565DA3D8', 'notBefore': 'Apr 14 19:04:49 2023 GMT', 'notAfter': 'Jul 13 19:04:48 2023 GMT', 'subjectAltName': (('DNS', 'cbrc3.cbrc.jp'),), 'OCSP': ('http://r3.o.lencr.org',), 'caIssuers': ('http://r3.i.lencr.org/',)}
  DEBUG:urllib3.util.retry:Incremented Retry for (url='/last-490.zip'): Retry(total=2, connect=None, read=None, redirect=None, status=None)
  WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLCertVerificationError("hostname 'last.cbrc.jp' doesn't match 'cbrc3.cbrc.jp'"))': /last-490.zip
  WARNING:urllib3.connection:Certificate did not match expected hostname: last.cbrc.jp. Certificate: {'subject': ((('commonName', 'cbrc3.cbrc.jp'),),), 'issuer': ((('countryName', 'US'),), (('organizationName', "Let's Encrypt"),), (('commonName', 'R3'),)), 'version': 3, 'serialNumber': '046FF5BB109D36E9CD09D5B800EC565DA3D8', 'notBefore': 'Apr 14 19:04:49 2023 GMT', 'notAfter': 'Jul 13 19:04:48 2023 GMT', 'subjectAltName': (('DNS', 'cbrc3.cbrc.jp'),), 'OCSP': ('http://r3.o.lencr.org',), 'caIssuers': ('http://r3.i.lencr.org/',)}
  DEBUG:urllib3.util.retry:Incremented Retry for (url='/last-490.zip'): Retry(total=1, connect=None, read=None, redirect=None, status=None)
  WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLCertVerificationError("hostname 'last.cbrc.jp' doesn't match 'cbrc3.cbrc.jp'"))': /last-490.zip
  WARNING:urllib3.connection:Certificate did not match expected hostname: last.cbrc.jp. Certificate: {'subject': ((('commonName', 'cbrc3.cbrc.jp'),),), 'issuer': ((('countryName', 'US'),), (('organizationName', "Let's Encrypt"),), (('commonName', 'R3'),)), 'version': 3, 'serialNumber': '046FF5BB109D36E9CD09D5B800EC565DA3D8', 'notBefore': 'Apr 14 19:04:49 2023 GMT', 'notAfter': 'Jul 13 19:04:48 2023 GMT', 'subjectAltName': (('DNS', 'cbrc3.cbrc.jp'),), 'OCSP': ('http://r3.o.lencr.org',), 'caIssuers': ('http://r3.i.lencr.org/',)}
  DEBUG:urllib3.util.retry:Incremented Retry for (url='/last-490.zip'): Retry(total=0, connect=None, read=None, redirect=None, status=None)
  WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLCertVerificationError("hostname 'last.cbrc.jp' doesn't match 'cbrc3.cbrc.jp'"))': /last-490.zip
  WARNING:urllib3.connection:Certificate did not match expected hostname: last.cbrc.jp. Certificate: {'subject': ((('commonName', 'cbrc3.cbrc.jp'),),), 'issuer': ((('countryName', 'US'),), (('organizationName', "Let's Encrypt"),), (('commonName', 'R3'),)), 'version': 3, 'serialNumber': '046FF5BB109D36E9CD09D5B800EC565DA3D8', 'notBefore': 'Apr 14 19:04:49 2023 GMT', 'notAfter': 'Jul 13 19:04:48 2023 GMT', 'subjectAltName': (('DNS', 'cbrc3.cbrc.jp'),), 'OCSP': ('http://r3.o.lencr.org',), 'caIssuers': ('http://r3.i.lencr.org/',)}
  Traceback (most recent call last):
    File "/opt/conda/lib/python3.8/site-packages/urllib3/connectionpool.py", line 670, in urlopen
      httplib_response = self._make_request(
    File "/opt/conda/lib/python3.8/site-packages/urllib3/connectionpool.py", line 381, in _make_request
      self._validate_conn(conn)
    File "/opt/conda/lib/python3.8/site-packages/urllib3/connectionpool.py", line 978, in _validate_conn
      conn.connect()
    File "/opt/conda/lib/python3.8/site-packages/urllib3/connection.py", line 397, in connect
      _match_hostname(cert, self.assert_hostname or server_hostname)
    File "/opt/conda/lib/python3.8/site-packages/urllib3/connection.py", line 407, in _match_hostname
      match_hostname(cert, asserted_hostname)
    File "/opt/conda/lib/python3.8/ssl.py", line 420, in match_hostname
      raise CertificateError("hostname %r "
  ssl.SSLCertVerificationError: ("hostname 'last.cbrc.jp' doesn't match 'cbrc3.cbrc.jp'",)

  During handling of the above exception, another exception occurred:

  Traceback (most recent call last):
    File "/opt/conda/lib/python3.8/site-packages/requests/adapters.py", line 439, in send
      resp = conn.urlopen(
    File "/opt/conda/lib/python3.8/site-packages/urllib3/connectionpool.py", line 754, in urlopen
      return self.urlopen(
    File "/opt/conda/lib/python3.8/site-packages/urllib3/connectionpool.py", line 754, in urlopen
      return self.urlopen(
    File "/opt/conda/lib/python3.8/site-packages/urllib3/connectionpool.py", line 754, in urlopen
      return self.urlopen(
    File "/opt/conda/lib/python3.8/site-packages/urllib3/connectionpool.py", line 726, in urlopen
      retries = retries.increment(
    File "/opt/conda/lib/python3.8/site-packages/urllib3/util/retry.py", line 446, in increment
      raise MaxRetryError(_pool, url, error or ResponseError(cause))
  urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='last.cbrc.jp', port=443): Max retries exceeded with url: /last-490.zip (Caused by SSLError(SSLCertVerificationError("hostname 'last.cbrc.jp' doesn't match 'cbrc3.cbrc.jp'")))

  During handling of the above exception, another exception occurred:

  Traceback (most recent call last):
    File "/opt/conda/lib/python3.8/site-packages/conda/gateways/connection/download.py", line 48, in download
      resp = session.get(url, stream=True, proxies=session.proxies, timeout=timeout)
    File "/opt/conda/lib/python3.8/site-packages/requests/sessions.py", line 546, in get
      return self.request('GET', url, **kwargs)
    File "/opt/conda/lib/python3.8/site-packages/requests/sessions.py", line 533, in request
      resp = self.send(prep, **send_kwargs)
    File "/opt/conda/lib/python3.8/site-packages/requests/sessions.py", line 668, in send
      history = [resp for resp in gen] if allow_redirects else []
    File "/opt/conda/lib/python3.8/site-packages/requests/sessions.py", line 668, in <listcomp>
      history = [resp for resp in gen] if allow_redirects else []
    File "/opt/conda/lib/python3.8/site-packages/requests/sessions.py", line 239, in resolve_redirects
      resp = self.send(
    File "/opt/conda/lib/python3.8/site-packages/requests/sessions.py", line 646, in send
      r = adapter.send(request, **kwargs)
    File "/opt/conda/lib/python3.8/site-packages/requests/adapters.py", line 514, in send
      raise SSLError(e, request=request)
  requests.exceptions.SSLError: HTTPSConnectionPool(host='last.cbrc.jp', port=443): Max retries exceeded with url: /last-490.zip (Caused by SSLError(SSLCertVerificationError("hostname 'last.cbrc.jp' doesn't match 'cbrc3.cbrc.jp'")))

  During handling of the above exception, another exception occurred:

  Traceback (most recent call last):
    File "/opt/conda/bin/conda-mambabuild", line 10, in <module>
      sys.exit(main())
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 256, in main
      call_conda_build(action, config)
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 228, in call_conda_build
      result = api.build(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/api.py", line 180, in build
      return build_tree(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 3078, in build_tree
      packages_from_this = build(metadata, stats,
    File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 2125, in build
      try_download(m, no_download_source=False, raise_error=True)
    File "/opt/conda/lib/python3.8/site-packages/conda_build/render.py", line 650, in try_download
      source.provide(metadata)
    File "/opt/conda/lib/python3.8/site-packages/conda_build/source.py", line 871, in provide
      unpack(source_dict, src_dir, metadata.config.src_cache, recipe_path=metadata.path,
    File "/opt/conda/lib/python3.8/site-packages/conda_build/source.py", line 146, in unpack
      src_path, unhashed_fn = download_to_cache(cache_folder, recipe_path, source_dict, verbose)
    File "/opt/conda/lib/python3.8/site-packages/conda_build/source.py", line 89, in download_to_cache
      download(url, path)
    File "/opt/conda/lib/python3.8/site-packages/conda/exports.py", line 374, in download
      return _download(url, dst_path, md5=md5sum, sha256=sha256, size=size)
    File "/opt/conda/lib/python3.8/site-packages/conda/common/io.py", line 84, in decorated
      return f(*args, **kwds)
    File "/opt/conda/lib/python3.8/site-packages/conda/gateways/connection/download.py", line 152, in download
      raise CondaSSLError(
  conda.exceptions.CondaSSLError: Encountered an SSL error. Most likely a certificate verification issue.

  Exception: HTTPSConnectionPool(host='last.cbrc.jp', port=443): Max retries exceeded with url: /last-490.zip (Caused by SSLError(SSLCertVerificationError("hostname 'last.cbrc.jp' doesn't match 'cbrc3.cbrc.jp'")))
